python/examples/multiSchema/agents.py

```diff
@@ -0,0 +1,106 @@
+import os
+import sys

```

Comment:
I don't have a strong opinion either way because it could be nice to just pass in a function and a string - but if you do choose to go the base class route, the base class should probably make no mention of the privates - `handle_request` and `agent_name` might be fine though.